src/scan_to_cloud_filter_chain.cpp

```diff
@@ -245,18 +264,28 @@ class ScanToCloudFilterChain
 
 } ;
 
+#if BUILDING_NODELET
 
+class ScanToCloudFilterChainNodelet : public nodelet::Nodelet
+{
+  std::unique_ptr<ScanToCloudFilterChain> chain_;
 
+  void onInit() override {
+    chain_ = std::make_unique<ScanToCloudFilterChain>(getPrivateNodeHandle(), getName());
+  }
+};
+
+PLUGINLIB_EXPORT_CLASS(ScanToCloudFilterChainNodelet, nodelet::Nodelet)
+#else
 int
 main (int argc, char** argv)
 {
   ros::init (argc, argv, "scan_to_cloud_filter_chain");
-  ros::NodeHandle nh;
-  ScanToCloudFilterChain f;
+  ros::NodeHandle nh, pnh("~");
+  ScanToCloudFilterChain f(pnh, "");
 
   ros::spin();
 
   return (0);
 }
-
-
+#endif
```

Comment:
Nitpick - could you add an end line to the last line of this file and nodelets.xml?